ui/pages/settings/snaps/snaps-list-tab/snap-list-tab.js

```diff
@@ -61,62 +70,71 @@ const SnapListTab = () => {
+      )}
+      {snapsList.length <= 5 && (
         <Box
-          className="snap-list-tab__container--no-snaps"
-          width="full"
-          height="full"
-          alignItems={AlignItems.center}
-          flexDirection={FLEX_DIRECTION.COLUMN}
+          display={Display.Flex}
+          height={BlockSize.Full}
+          flexDirection={FlexDirection.Row}
+          flexWrap={FlexWrap.Wrap}
+          justifyContent={JustifyContent.center}
+          className="snap-list-tab__container--snaps-info-content"
         >
-          <Box
-            className="snap-list-tab__container--no-snaps_inner"
-            width="full"
-            height="full"
-            flexDirection={FLEX_DIRECTION.COLUMN}
-            justifyContent={JustifyContent.center}
-            alignItems={AlignItems.center}
-          >
-            <Icon
-              name={IconName.Snaps}
-              color={IconColor.iconMuted}
-              className="snap-list-tab__no-snaps_icon"
-              size={IconSize.Inherit}
-            />
-            <Text
-              color={Color.textMuted}
-              align={TEXT_ALIGN.CENTER}
-              marginTop={4}
+          {snapsList.length < 1 && (
+            <Box
+              className="snap-list-tab__container--no-snaps_inner"
+              display={Display.Flex}
+              flexDirection={FlexDirection.Column}
+              justifyContent={JustifyContent.center}
+              alignItems={AlignItems.center}
             >
-              {t('noSnaps')}
-            </Text>
-          </Box>
+              <Icon
+                name={IconName.Snaps}
+                color={IconColor.iconMuted}
+                className="snap-list-tab__no-snaps_icon"
+                size={IconSize.Inherit}
+              />
+              <Text
+                color={Color.textMuted}
+                align={TextAlign.Center}
+                marginTop={4}
+              >
+                {t('noSnaps')}
+              </Text>
+            </Box>
+          )}
+          <Box
+            display={Display.Flex}
+            width={BlockSize.Full}
+            height={BlockSize.Min}
+          ></Box>
           <Box
             className="snap-list-tab__container--no-snaps_banner-tip"
-            width="full"
-            justifyContent={JustifyContent.center}
-            alignItems={AlignItems.flexEnd}
+            display={Display.Flex}
+            flexDirection={FlexDirection.Column}
+            justifyContent={JustifyContent.flexEnd}
             paddingLeft={4}
             paddingRight={4}
             paddingBottom={4}
           >
             <BannerTip
               logoType={BannerTipLogoType.Greeting}
-              title={t('exploreMetaMaskSnaps')}
               description={t('extendWalletWithSnaps')}
+              descriptionProps={{ variant: TextVariant.bodyMd }}
             >
               <ButtonLink
                 size={Size.auto}
                 href="https://metamask.io/snaps/"
```

Comment:
Should this link to the directory instead?

Reply:
Honestly, I don't know. I never clearly received any instructions that this should be linking directly to the directory instead of the main Snaps website 🤔
Directory is like "two clicks away" currently.
@eriknson @Montoya

Reply:
This will not link to the directory *yet*. Current approach is correct.